internal/services/compute/migration/managed_disk_v0_to_v1.go

```diff
@@ -197,7 +201,7 @@ func (ManagedDiskV0ToV1) Schema() map[string]*pluginsdk.Schema {
 	}
 }
 
-func encryptionSettingsSchema() *pluginsdk.Schema {
```

Comment:
I missed this in the initial PR, but since this is a point in time reference we need to remove the schema feature flag and the Schema for 4.0.

Reply:
make sense. I've removed the feature flag and kept only the 3.0 schema for this